.github/actions/system-info/system-info.py

```diff
@@ -0,0 +1,32 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+import platform
+import sys
+
+print(f"{sys.platform=}", file=sys.stderr)
```

Comment:
Just curious: why do we need to print to `sys.stderr`?

Reply:
To not "poison" `$GITHUB_OUTPUT`